src/cf-layout/src/organisms/featured-content-module.less

```diff
@@ -0,0 +1,101 @@
+
+@fcm-bp-max: 700px;
+@fcm-bp-min: 701px;
+
+.o-featured-content-module {
+  .webfont-regular();
+
+  position: relative;
+
+  min-height: 320px;
+  margin-right: -1px;
+  margin-left: -1px;
+
+  background-color: @block__bg;
+
+  .respond-to-max( @bp-xs-max, {
+    padding-right: 15px;
+    padding-left: 15px;
+  });
+
+  .respond-to-range( @bp-sm-min, @fcm-bp-max, {
+    padding-right: 30px;
+    padding-left: 30px;
+  });
+
+  &_text {
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em);
+    padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em);
+
+    .respond-to-min( @fcm-bp-min, {
+      width: 300px; // width of 5 of 12 cols at 799px - 30px padding
+
+      padding-right: unit( @grid_gutter-width / @base-font-size-px, em);
+      padding-left: unit( @grid_gutter-width / @base-font-size-px, em);
+    });
+
+    .respond-to-min( @bp-med-min, {
+      width: 270px; // width of 3 of 12 cols at 1140px - 15px padding
+    });
+  }
+
+  &_visual {
+    .respond-to-min( @fcm-bp-min, {
+      position: absolute;
+      left: 360px;
+      right: 0;
+      top: 0;
+      bottom: 0;
+      overflow: hidden;
+    });
+
+    .respond-to-min( @bp-med-min, {
+      left: 330px;
+    });
+  }
+
+  &_img {
+    display: block;
+
+    .respond-to-min( @fcm-bp-min, {
+      height: 100%;
+      max-width: none;
+
+      .lt-ie9 & {
+        left: -302.5px; // Width at full width
+        margin-left: 50%;
+      }
+    });
+
+    &.o-featured-content-module_img__map {
```

Comment:
Instead of `map` and `video` should we use `right` and `center` modifiers to keep it future proof when someone inevitably has another use case, like a specific photo or illustration that just happens to be oriented differently?